src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -4,9 +4,10 @@ import {Text, View, StyleSheet, TouchableOpacity, Pressable, Platform} from 'rea
 import ToastMessage from 'react-native-toast-message';
 import TextAreaChat from '../../../components/TextAreaChat';
 import ToggleSwitch from '../../../components/ToggleSwitch';
-import {colors} from '../../../utils/colors';
 import {profileSettingsDMpermission} from '../../../service/profile';
 import {addDotAndRemoveNewline} from '../../../utils/string/TrimString';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use the new `COLORS` object from the `theme` module instead of the old `colors` module. This is consistent with the changes made across the application.
```diff
- import {colors} from '../../../utils/colors';
+ import {COLORS} from '../../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->